distributions/standalone/assembly-whitelist.txt

```diff
@@ -154,6 +154,7 @@
 /mule-standalone-${productVersion}/lib/mule/mule-transport-sockets-${productVersion}.jar
 /mule-standalone-${productVersion}/lib/mule/mule-module-tls-${productVersion}.jar
 /mule-standalone-${productVersion}/lib/mule/mule-module-tomcat-${productVersion}.jar
+/mule-standalone-${productVersion}/lib/mule/mule-module-tooling-${productVersion}.jar
```

Comment:
this module name seems weird... tooling-support might be more consistent, but things like connectivity testing are probably not exclusive to tooling.